platform/windows/detect.py

```diff
@@ -190,7 +190,7 @@ def configure(env):
         if env["bits"] == "64":
             env.Append(CCFLAGS=['/D_WIN64'])
 
-        LIBS = ['winmm', 'opengl32', 'dsound', 'kernel32', 'ole32', 'oleaut32', 'user32', 'gdi32', 'IPHLPAPI', 'Shlwapi', 'wsock32', 'Ws2_32', 'shell32', 'advapi32', 'dinput8', 'dxguid']
+        LIBS = ['winmm', 'opengl32', 'dsound', 'kernel32', 'ole32', 'oleaut32', 'user32', 'gdi32', 'IPHLPAPI', 'Shlwapi', 'wsock32', 'Ws2_32', 'shell32', 'advapi32', 'dinput8', 'dxguid', 'Setupapi']
```

Comment:
Linking fails while cross-compiling from Unix:
```
x86_64-w64-mingw32-g++ -o bin/godot.windows.tools.64.exe -static platform/windows/godot_win.windows.tools.64.o platform/windows/context_gl_win.windows.tools.64.o platform/windows/os_windows.windows.tools.64.o platform/windows/ctxgl_procaddr.windows.tools.64.o platform/windows/key_mapping_win.windows.tools.64.o platform/windows/tcp_server_winsock.windows.tools.64.o platform/windows/packet_peer_udp_winsock.windows.tools.64.o platform/windows/stream_peer_winsock.windows.tools.64.o platform/windows/joypad.windows.tools.64.o platform/windows/power_windows.windows.tools.64.o platform/windows/godot_res.windows.tools.64.o main/libmain.windows.tools.64.a main/tests/libtests.windows.tools.64.a modules/libmodules.windows.tools.64.a drivers/libdrivers.windows.tools.64.a editor/libeditor.windows.tools.64.a scene/libscene.windows.tools.64.a servers/libservers.windows.tools.64.a core/libcore.windows.tools.64.a modules/freetype/libfreetype_builtin.windows.tools.64.a -lmingw32 -lopengl32 -ldsound -lole32 -ld3d9 -lwinmm -lgdi32 -liphlpapi -lshlwapi -lwsock32 -lws2_32 -lkernel32 -loleaut32 -ldinput8 -ldxguid -lSetupapi
/usr/bin/x86_64-w64-mingw32-ld: cannot find -lSetupapi
collect2: error: ld returned 1 exit status
```

I guess it should be `setupapi`:
```
# find /usr/i686-w64-mingw32/ | grep -i setupapi
/usr/i686-w64-mingw32/sys-root/mingw/lib/libsetupapi.a
/usr/i686-w64-mingw32/sys-root/mingw/include/setupapi.h
```